### PR TITLE
HW  Library Completed

### PR DESCRIPTION
**Pull request by GKundu**
@author Goutam Kundu & Francisco Latacela

### CODE REVIEW

**Comment by khatchad** [comment]
I don't have a Blackboard submission from your group. Please note the submission instructions and submit the GitHub username of the account you'd like me to grade ASAP:

> ... 
> 1. [Create a **pull request**](https://help.github.com/articles/creating-a-pull-request) on the original repository to turn in the assignment.
> 2. Submit your GitHub **username** on [Blackboard](https://bbhosted.cuny.edu/webapps/assignment/uploadAssignment?content_id=_20246794_1&course_id=_1101235_1&assign_group_id=&mode=cpview).

**Comment by GKundu** [comment]
Hello Prof,

I have updated the information on blackbord as you requested ; thanks for getting back to me .

User name - gkundu for lab 7

File: Library/src/lab1/Book.java [hunk 1]
```diff
@@ -18,50 +19,58 @@
...
+     *
      * @return This book's title as a String.
      */
     public String getTitle() {
-        // TODO: Implement this method
+        return title;
     }
 
     public static void main(String[] arguments) {
-        // Small test of the Book class
```
Comment (khatchad) [inline Library/src/lab1/Book.java:58]: :-1: Per the instructions:

> You must _not_ modify the main method.

File: Library/src/lab1/Book.java [hunk 2]
```diff
@@ -18,50 +19,58 @@
...
     }
-   
+
     /**
      * Marks this book as not rented.
      */
     public void returned() {
-        // TODO: Implement this method
+        borrowed = false;
+        System.out.println("This book is not rented, you can take it");
```
Comment (khatchad) [inline Library/src/lab1/Book.java:43]: :-1: This method should not have any output.

**Comment by khatchad** [comment]
:-1: Expected:

> Borrowing The Lord of the Rings:
> You successfully borrowed The Lord of the Rings
> Sorry, this book is already borrowed.
> Sorry, this book is not in our catalog.

Actual:

> Borrowing The Lord of the Rings:
> **Sorry, This book is rented**
> You successfully borrowed The Lord of the Rings
> Sorry, this book is already borrowed.
> Sorry, this book is not in our catalog.

File: Library/src/lab1/Library.java [hunk 3]
```diff
@@ -2,11 +2,85 @@
...
+                presentBook = true;
+            }
+        }
+        if (!presentBook) {
+            System.out.println("No book in catalog");
+        }
+    }
+
+    public void borrowBook(String title) {
+        int foundBook = 0;
```
Comment (khatchad) [inline Library/src/lab1/Library.java]: `boolean`

File: Library/src/lab1/Library.java [hunk 4]
```diff
@@ -2,11 +2,85 @@
...
+        for (Book book : books) {
+            if (book.getTitle().equals(title)) {
+                if (foundBook == 0) {
```
Comment (khatchad) [inline Library/src/lab1/Library.java:51]: Why not just set it to 1?

**Comment by khatchad** [comment]
:+1:

**Pull request closed without merging**